test/src/test/java/org/corfudb/integration/CorfuReplicationLeaderIT.java

```diff
@@ -0,0 +1,730 @@
+                log.info("Standby table size: {}", replicationTableStandby.size());
+                Sleep.sleepUninterruptibly(mainThreadSleepDuration);
+            }
+            backGroundWriter.join();
+        } finally {
+            cleanUpResources(
+                    ImmutableList.of(activeRuntime, standbyRuntime),
+                    Stream.concat(primaries.stream(), standBys.stream()).map(t -> t.second)
+                            .collect(ImmutableList.toImmutableList()),
+                    ImmutableList.of(corfuPrimary, corfuStandBy));
+        }
+
+    }
+
+
+    @Test
+    public void testLockHolderContinuouslyFlappingSource() throws Exception {
+        final int numWriteIterations = 20;
+        final Duration waitBetweenWrites = Duration.ofSeconds(2);
+        final Duration mainThreadSleepDuration = Duration.ofSeconds(1);
+        final Duration durationBetweenFlaps = Duration.ofSeconds(10);
+        final int flaps = 2;
+
+        String pluginConfigPath = "src/test/resources/topology/two_primaries.properties";
+        String configPath = "disable_lock_config.properties";
+        ImmutableList<Tuple<UUID, CorfuInterClusterReplicationServer>> primaries =
+                TWO_PRIMARY_REPLICATION_SERVER_PORTS.stream()
+                        .map(port -> runReplicationFuture(port, pluginConfigPath, configPath))
+                        .collect(ImmutableList.toImmutableList());
+
+        ImmutableList<Tuple<UUID, CorfuInterClusterReplicationServer>> standBys =
+                ONE_STANDBY_REPLICATION_SERVER_PORT.stream()
+                        .map(port -> runReplicationFuture(port, pluginConfigPath, configPath))
+                        .collect(ImmutableList.toImmutableList());
+
+        CorfuRuntime.CorfuRuntimeParameters params = CorfuRuntime.CorfuRuntimeParameters
+                .builder()
+                .build();
+
+        CorfuRuntime activeRuntime = CorfuRuntime
+                .fromParameters(params)
+                .setTransactionLogging(true)
+                .parseConfigurationString(PRIMARY_CORFU_ENDPOINT);
+
+        CorfuRuntime standbyRuntime =
+                CorfuRuntime.fromParameters(params)
+                        .setTransactionLogging(true)
+                        .parseConfigurationString(STANDBY_CORFU_ENDPOINT);
+        try {
+
+            activeRuntime = activeRuntime.connect();
+            standbyRuntime = standbyRuntime.connect();
+
+            CorfuTable<String, Integer> replicationTableStandby = standbyRuntime.getObjectsView()
+                    .build()
+                    .setStreamName(REPLICATION_TABLE_NAME)
+                    .setTypeToken(new TypeToken<CorfuTable<String, Integer>>() {
+                    })
+                    .open();
+
+            CompletableFuture<Void> backGroundWriter =
+                    runBackGroundWriter(numWriteIterations, waitBetweenWrites);
+            final int third = 3;
```

Comment:
what does this third mean/represent?

Reply:
Its just a constant for a fraction. Checkstyle complains if we don't use constants. I thinks its called magic number or something